sparkstreaming-kinesis/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2016-2017 The OpenZipkin Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>zipkin-aws-parent</artifactId>
+        <groupId>io.zipkin.aws</groupId>
+        <version>0.1.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>sparkstreaming-kinesis-parent</artifactId>
+    <packaging>pom</packaging>
+
+    <properties>
+        <main.basedir>${project.basedir}/..</main.basedir>
+    </properties>
+
+    <modules>
+        <module>stream</module>
+        <module>autoconfigure</module>
```

Comment:
So I honestly felt it made it easier to work on with the 2 related modules as siblings. I'm not quite sure if that is actually the case. I can move the autoconfigure module into the existing zipkin-aws/autoconfigure module since that will probably be easier for an outsider to understand.

I also feel that modules for different dependecies might be easier to maintain. I'm not quite sure if thats a good topic for this particular PR though, maybe thats a bigger discussion to have with others involved in this repo and others like it (azure)